src/cli/commands/init.js

```diff
@@ -114,14 +121,18 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       def = val;
     }
 
+    if (manifestKey === 'private' && privateFlag) {
```

Comment:
This conditional could be eliminated if the default is set above in the array of questions